public/js/components/Editor.css

```diff
@@ -27,23 +27,28 @@
   width: 60px;
   height: 12px;
   position: absolute;
-  top: 0;
+  top: 1px;
   right: -4px;
 }
 
 .editor.new-breakpoint.breakpoint-disabled svg {
   opacity: 0.3;
 }
 
+.search-bar ~ .editor-mount {
+  height: calc(100% - 58px)
+}
+
 .editor-mount,
 .CodeMirror {
   width: 100%;
-  height: 100%;
+  height: calc(100% - 17px);
```

Comment:
Yeah, that makes sense. Would probably be good to pull this out into a separate PR with a real fix.

I haven't looked at this CSS for a while, but seems like we could use flexbox and tell the editor-container to fill up as much as possible, while the header and footer take up as little as possible (`flex: 0`). But not sure.